src/index.html

```diff
@@ -132,32 +96,4 @@ <h2>Read more</h2>
       {% endfor %}
     </div>
   </div>
-
-  <div class="usa-width-one-whole">
-    <br>
-    <p class="page-width-70">
-      Some photos of us doing things we find amusing.
-    </p>
-    <br>
-  </div>
-
-  <div class="dds-grid">
-    {% for photo in site.data.gallery %}
-    <div class="dds-team-item">
-      <div class="box">
-        <a href="{{ photo.full }}">
-          <img class="photo" src="{{ photo.thumb }}" alt="{{ photo.description }} ({{ photo.credit}})" title="({{ photo.credit }})">
-        </a>
-      </div>
-    </div>
-    {% endfor %}
-  </div>
-</section>
-
-<section id="image_disclaimer" class="usa-grid">
-  <div class="usa-width-one-whole">
-    <div class="page-width-70">
-      See the <a href="/privacy.html#image-rights">Privacy Policy</a> for image credits and information on reusing images.
```

Comment:
This part is important, in my opinion. We may need to get @willgov's take on removing this. It was put in by Sharon long ago, and intentionally.

Reply:
I removed this cause I removed the photos section, but I guess there's still the header image? I can add it back in

Reply:
Yeah, and if we replace the header image then we may still need this. Also, are we deleting the photos themselves in the repo? If so, we can eliminate those lines in the privacy policy file maybe? Should check with @willgov

Reply:
I talked to Will about this and he said it's not necessary to have this line explicitly on our homepage as long as we have a link to this information somewhere (it's in the footer). 

Also, we can should separate this out from the Privacy Policy since it's not privacy-related, and change it to Intellectual Property Information. It's also not necessary to list every image in that section, as long as we have some generic language about government IP (already there), and add copyright info for non-govt owned images with the image as a caption or in some other form. That seems easier to maintain

Reply:
Cool beans